### PR TITLE
Assessment about input focus and add assets support for a11y report

### PR DESCRIPTION
**Pull request by duboisp**
This is a content change

* Assessment about input focus 
* Add support to display assets in our web accessibility assessment report
* Add the i18n text for displaying the report section on the main GCWeb page

cc @shawnthompson

### CODE REVIEW

File: _data/i18n.yml [hunk 1]
```diff
@@ -7,13 +7,15 @@ page_group:
...
     test: Tests pages
+    reports: Reports
   fr:
     examples: Exemples
     docs: Documentations
     related: Documents relié
     test: Pages d'essaie
     impl-ref: Référence d'implémentation
     spec: Spécification
+    reports: Raports
```
Comment (Garneauma) [inline _data/i18n.yml:18]:
```suggestion
    reports: Rapports
```

File: common/scaffolding/reports/a11y-1.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    "acr:standard": "https://www.w3.org/TR/WCAG21",
+    "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+    "earl:result": [
+        {
+            "@id": "_:sc_1.4.1",
+            "earl:test": "WCAG21:non-text-contrast",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:failed",
+            "dct:description": "Color contrast is not enough when an input field receive the focus. The contrast measured is: 2.2:1",
```
Comment (Garneauma) [inline common/scaffolding/reports/a11y-1.json:39]:
```suggestion
            "dct:description": "Color contrast is not enough when an input field receives the focus. The contrast measured is: 2.2:1",
```

File: common/scaffolding/reports/a11y-1.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            "dct:modified": "2023-11-21",
+            "acr:asset": [
+              {
+                "@type": "acr:AttachmentImage",
+                "dct:title": "Visual capture of the input element when it receive focus\nContains the following text: Title saying 'The (code) input (end code) element'; Sub heading saying 'Implicit labeling'; Label saying 'Label:'; an input field with a blue focus border; Code sample of the preceding which say '<form><label>Label: <input></label>'",
```
Comment (Garneauma) [inline common/scaffolding/reports/a11y-1.json:44]:
```suggestion
                "dct:title": "Visual capture of the input element when it receives focus\nContains the following text: Title saying 'The (code) input (end code) element'; Sub heading saying 'Implicit labeling'; Label saying 'Label:'; an input field with a blue focus border; Code sample of the preceding which says '<form><label>Label: <input></label>'",
```

**Review by Garneauma — CHANGES_REQUESTED** [review]
Simples petits changements textuels.

**Review by Garneauma — APPROVED** [review]

**Comment by duboisp** [comment]
@shawnthompson here the permanent link of your assessment: https://wet-boew.github.io/GCWeb/common/scaffolding/reports/a11y-1-en.html

**Pull request merged**